scripts/build-local-release.sh

```
@@ -0,0 +1,125 @@
+  case "$arg" in
+    --sync) SYNC=1 ;;
+    -h|--help)
+      echo "Usage: $0 [--sync]"
+      echo "  --sync  Fetch upstream/main and rebase feature branch before building"
+      exit 0
+      ;;
+  esac
+done
+
+# --- Signing configuration ---
+SIGN_IDENTITY="Developer ID Application: Emily Carr University of Art and Design (7TF6CSP83S)"
```

Shellcheck (0.11.0)

[warning] SIGN_IDENTITY appears unused. Verify use (or export if used externally).

(SC2034)